code/modules/organs/internal/brain.dm

```diff
@@ -99,13 +99,13 @@
 	alert(owner, "You have taken massive brain damage! You will not be able to remember the events leading up to your injury.", "Brain Damaged")
 
 /obj/item/organ/internal/brain/organ_can_heal()
-	return (damage && owner && GET_CHEMICAL_EFFECT(owner, CE_BRAIN_REGEN)) || ..()
+	return (damage && owner && GET_CHEMICAL_EFFECT(owner, CE_BRAIN_REGEN) > 0) || ..()
 
 /obj/item/organ/internal/brain/has_limited_healing()
-	return (!owner || !GET_CHEMICAL_EFFECT(owner, CE_BRAIN_REGEN)) && ..()
+	return (!owner || GET_CHEMICAL_EFFECT(owner, CE_BRAIN_REGEN) <= 0) && ..()
 
 /obj/item/organ/internal/brain/get_organ_heal_amount()
-	return 1
+	return owner ? GET_CHEMICAL_EFFECT(owner, CE_BRAIN_REGEN) : 1
```

Comment:
oh, so it *is* intended that it never heals on its own if you don't have fish oil? otherwise you should probably be adding 1 to the effect

Reply:
well, I guess thats a change to request?
```suggestion
	return 1 + (owner ? GET_CHEMICAL_EFFECT(owner, CE_BRAIN_REGEN) : 0)
```
this will mean it heals 1 base, and then gains more based on the regen effect strength.